windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -175,12 +175,13 @@ cp $LIB_GCC_DLL $SD_LOGGING_AGENT_DIR_BIN
 #  STEP 5.1 - TEMPORARY HACK TO UPDATE RUBY FILE
 ##############################
 # 
-# TODO: Remove when https://github.com/djberg96/win32-eventlog/pull/23 is merged.
+# TODO: Remove when https://github.com/djberg96/win32-eventlog/pull/23 is merged and released.
+# TODO: Remove when https://github.com/djberg96/win32-eventlog/pull/24 is released.
 ##############################
 
 $eventlog_rb = $SD_LOGGING_AGENT_DIR + '\lib\ruby\gems\2.3.0\gems\win32-eventlog-0.6.6\lib\win32\eventlog.rb'
 $needle = 'max_insert = [num, buf.read_string.scan(/%(\d+)/).map{ |x| x[0].to_i }.max].compact.max'
```

Comment:
But we'll still need to update $needle, won't we, if PR#24 is released, but PR#23 isn't merged yet?